app/assets/stylesheets/_defaults.scss

```diff
@@ -1,13 +1,47 @@
+@import 'https://fonts.googleapis.com/css?family=Slabo+27px';
 @import "hocus-pocus/scss/colors";
 
+$rwd-type: desktop-first;
+$rwd-map: (mobile: 480px, tablet: 720px, desktop: 1201px);
```

Comment:
a bit unusual breakpoints, why 1201px for desktop? just want to make sure it was tested